x-pack/plugins/observability/public/utils/no_data_config.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { IBasePath } from '../../../../../src/core/public';
+import { KibanaPageTemplateProps } from '../../../../../src/plugins/kibana_react/public';
+
+export function getNoDataConfig({
+  docsLink,
+  basePath,
+  hasData,
+}: {
+  docsLink: string;
+  basePath: IBasePath;
+  hasData?: boolean;
+}): KibanaPageTemplateProps['noDataConfig'] {
+  if (hasData === false) {
+    return {
```

Comment:
nit: you can reduce the indentation like

```suggestion
  if (hasData !== false) {
    return undefined;
  }
  
  return {
```